### DISCUSSION TITLE
Icon Request: fa-blueprint

### DISCUSSION THREAD

**Discussion by mdolberg** [FortAwesome/Font-Awesome#21189] [Icon Requests]
I am planning to use it as a way to express construction

![blueprint](https://cloud.githubusercontent.com/assets/11992836/10965073/420fef32-83b3-11e5-9252-bfa08630104b.png)

**Comment by lenamtl** [comment]
+1

**Comment by danielklotz** [comment]
+1

**Comment by wiebedijkstra** [comment]
+1

**Comment by dmbrenna** [comment]
+1

**Comment by austinfelipe** [comment]
+1

**Comment by Relequestual** [comment]
I don't see this in the list of requested icons on the FA site. Any idea why that could be?

**Comment by eqMFqfFd** [comment]
+1

**Comment by eqMFqfFd** [comment]
5 years later?

**Comment by gdvalderrama** [comment]
The one proposed here looks more complex, like two icons overlapped really.

I'm happy even with a simpler icon version, something like a square with an opening representing a door, i.e: https://icon-library.com/images/floor-plan-door-icon/floor-plan-door-icon-16.jpg

**Comment by alvarovmz** [comment]
Too complex. @gdvalderrama solution is awesome

**Comment by dmartinezzm** [comment]
Please add it

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
